src/views/scroll-editor/artefact-image-group.vue

```diff
@@ -14,24 +14,57 @@
         <defs>
             <path :id="`path-${artefact.id}`" :d="artefact.mask.svg" />
             <clipPath :id="`clip-path-${artefact.id}`">
-                <use stroke="none" fill="black" fill-rule="evenodd" :href="`#path-${artefact.id}`" />
+                <use
+                    stroke="none"
+                    fill="black"
+                    fill-rule="evenodd"
+                    :href="`#path-${artefact.id}`"
+                />
             </clipPath>
         </defs>
         <g @click="onSelect">
-            <g :clip-path="`url(#clip-path-${artefact.id})`">
+            <g :clip-path="`url(#clip-path-${artefact.id})`" v-if="!artefact.isVirtual">
                 <iiif-image
                     :image="masterImage"
                     :boundingBox="boundingBox"
                     :scaleFactor="scaleFactor"
                 />
             </g>
+            <path v-if="artefact.isVirtual"
```

Comment:
Something looks wrong to me here.  In this path element and the one below it you check if `artefact.isVirtual`, but both elements are exactly the same except for one being of class `virtual-artefact` and the other being `virtual`.  Is there any reason you need two elements?

Reply:
I don't see why we need two. I'll confirm with Shaindel. Good catch!